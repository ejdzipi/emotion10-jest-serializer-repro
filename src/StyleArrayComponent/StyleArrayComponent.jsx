import React from 'react'
import {css} from '@emotion/core'

const noArrayStyle = css`
    display: flex;
`

const style1 = css`
    background-color: black;
`

const style2 = css`
    color: white;
`

export default () => {
    return (
        <div css={noArrayStyle}>
            Test content
            <div css={[style1, style2]}/>
        </div>
    );
};
