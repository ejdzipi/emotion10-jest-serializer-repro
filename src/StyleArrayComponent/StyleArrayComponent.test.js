import React from 'react'
import {mount} from 'enzyme'
import StyleArrayComponent from "./StyleArrayComponent";
import toJson from "enzyme-to-json";

describe('StyleArrayComponent', () => {
    it('enzyme - snapshot', () => {
        const component = mount(
            <StyleArrayComponent/>
        )

        expect(toJson(component)).toMatchSnapshot()
    })
})
