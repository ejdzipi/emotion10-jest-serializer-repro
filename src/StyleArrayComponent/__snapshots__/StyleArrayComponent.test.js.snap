// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyleArrayComponent enzyme - snapshot 1`] = `
.emotion-1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-0 {
  background-color: black;
  color: white;
}

<_default>
  <div
    className="emotion-1"
  >
    Test content
    <div
      className="emotion-0"
    />
  </div>
</_default>
`;
